### DISCUSSION TITLE
Монохромная панель 960*320

### DISCUSSION THREAD

**Discussion by IRBIS1630** [board707/DMD_STM32#39] [Ideas]
Мне понравилась ваша работа! Есть предложение и возможность для расширении вашей библиотеки на Github. Если интересно свяжитесь со мной. Буду рад вашему ответу!

**Comment by board707** [comment]
Здравствуйте.
Пишите прямо в дискуссию

**Comment by IRBIS1630** [comment]
Есть панели P10 960*320 хочу соединить их при помощи  вашей библиотеки с ПЛК 110 (к примеру)  по RS485. Верхние панели отображаю "Текст" (редко будут меняться), нижние "Цифры" (постоянно изменяются). ПЛК фирмы ОВЕН. Формат Ворд и стринг. Адрес который сам буду ставить с возможностью менять его. Я конечно не силен в программировании ("0") но надеюсь на вашу помощь. Контроллер LEd P10+ stm32+ RS845+PLK все это есть у меня есть возможность его проверить.

**Comment by board707** [comment]
Добрый день
Правильно ли я понял, что вы хотите управлять экраном по RS485, посылая ему команды для смены картинки на дисплее? В этом случае кто именно будет посылать команды - ПЛК или компьютер или другая ардуина - не имеет значения.
Уточните следующие моменты:
 - Какой общий размер экрана в пикселях?
 - Какие светодиодные панели используются - размер (пиксели), тип - монохром или цветные?
 - Какой формат сообщений? слова "Формат Ворд и стринг. " - не вполне понятны

Ну и собственно - что именно Вы хотите от меня и библиотеки. В первым сообщении Вы написали, что у Вас предложение по расширению библиотеки. В чем это будет заключаться?

**Reply by IRBIS1630** [reply]
Какой общий размер экрана в пикселях- P10 960 320 размер матрицы
Какие светодиодные панели использовать - размер (пиксели 10 mm), тип - монохромные
Сделать под эту библиотеку.

**Reply by board707** [reply]
Не совсем понял вас - у вас что, один модуль размером 96х32 см? - таких, по-моему, не бывает.
Или все-таки у вас 6 модулей 32х16, из которых вы составили панель 3х2 ?

Что касается задачи, напишите в почту dd@jwee.ru

**Reply by IRBIS1630** [reply]
6 модулей 32х16
